### ISSUE TITLE
linting system won't work for translators

### ISSUE THREAD

**Issue by samuelstroschein** [inlang/inlang#1139]
## Problem

- nobody uses the lint settings feature 
- lints are not adjustable by apps (turning on, off, error level, etc.)
- lint rules will likely always be provided by plugins 

The following API is not CRUD operable because it's code and will always rely on developers to change lint settings. Hint: Translators and PMs will hate this design.

https://github.com/inlang/inlang/blob/4ce4c35d35cf11e5bf10a06af09471d20749f022/source-code/core/src/lint/createLintRule.test-d.ts#L30-L45

## Proposal 

- remove lint settings feature 
- set lint level via simple id-level object that can be CRUD by apps 

Concept API for #1140 

```json
{
  "lint": {
    "rules": {
       "inlang.missingMessage": "off"
    }
  }

```

**Comment by samuelstroschein** [comment]
Further observations: the `createLintRule` function serves no benefit because (assumption) everyone will use `createPlugin`. 

- the main reason for `createLintRule` is the option to have scoped state and settings
- `createPlugin` serves the same purpose. 
- a lint rule could be vastly simplified by just saying "write a plugin that exposes a lint rule type"

**Comment by samuelstroschein** [comment]
More observations: If a simple CRUD config to set the level of lint rules exists, than it totally makes sense to bundle standard lint rules in core by default. 

Users can (via apps even!) turn off "annoying" standard lint rules.

**Comment by NilsJacobsen** [comment]
🤔 I'm not sure if that is the biggest pull factor for me to see CRUD as a benefit, but we can keep the door open with the conversion to JSON

**Comment by samuelstroschein** [comment]
will be addressed in #1149
